selfdrive/pandad/panda_safety.cc

```diff
@@ -56,16 +57,20 @@ std::string PandaSafety::fetchCarParams() {
   if (!params_.getBool("ControlsReady")) {
     return {};
   }
-  return params_.get("CarParams");
+  return {params_.get("CarParams"), params_.get("CarParamsSP")};
```

Comment:
I think I would feel better if this was a struct and not an array, this ain't python xD